internal/backend/client.go

```diff
@@ -27,6 +37,28 @@ func (c *clientProperties) action(entity, requestType string, body interface{})
 	return req.Make()
 }
 
+func (c clientProperties) actionWithRetry(entity, requestType string, body interface{}) request.Response {
```

Comment:
`func (c clientProperties)` should be `func (c *clientProperties)`

Reply:
updated